src/inference/src/dev/core_impl.hpp

```diff
@@ -250,8 +253,9 @@ class CoreImpl : public InferenceEngine::ICore, public std::enable_shared_from_t
      * @brief Register plugins for devices which are located in .xml configuration file.
      * @note The function supports UNICODE path
      * @param xmlConfigFile An .xml configuraion with device / plugin information
+     * @param ByAbsPath A boolean value - register plugins by absolute file path or not
      */
-    void register_plugins_in_registry(const std::string& xmlConfigFile);
+    void register_plugins_in_registry(const std::string& xmlConfigFile, const bool& ByAbsPath = false);
```

Comment:
Please use snake_case
```suggestion
    void register_plugins_in_registry(const std::string& xmlConfigFile, bool by_absolute_path = false);
```

Reply:
Are mixed types OK here? I mean `xmlConfigFile` and `by_absolute_path`

Reply:
You can change all names :)

Reply:
I will do it in next PR